app/assets/stylesheets/modules/mixins.scss

```diff
@@ -38,3 +38,17 @@
   border-color: #008080;
   color: #008080;
 }
+
+@mixin flash {
+  padding: 1em;
+  font-size: 1.2em;
+  font-weight: normal;
+  text-transform: uppercase;
+  line-height: 1;
+  letter-spacing: 3px;
+  text-align: center;
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.